views/js/runner/provider/qti.js

```diff
@@ -175,9 +175,9 @@ define([
             /**
              * Compute the next item for the given action
              * @param {String} action - item action like move/next, skip, etc.
-             * @param {Object} [params] - the item action additionnal params
+             * @param {Object} [params] - the item action additional params
              */
-            var computeNext = function computeNext(action, params){
+            function computeNext(action, params){
```

Comment:
This is related to ESLint errors fixing. Using variables functions and calling them before they are declared make ESLint unhappy.
In this code there is no need to use variable functions, as there is no reassignment. So I preferred to go to simple declaration instead of moving them all along to make the errors disappear.